doc/sailfish-webview-webview.qdoc

```diff
@@ -66,6 +68,505 @@
+  \brief Clears any text currently selected by the user.
+*/
+
+/*!
+  \qmlsignal WebView::linkClicked(string url)
+  \brief This signal is emitted when the user selects a link on the page.
+
+  When emitted \a url will be set to href selected.
+*/
+
+// ------------------------------------------------------ from RawWebView.qml:
+
+/*!
+  \qmlproperty qreal WebView::virtualKeyboardMargin
+  \brief The margin taken by the keyboard at the bottom of the page
+
+  When the keybaord is opened it may overlap web content displayed in a
+  \l{WebView}. Setting this value allows the web content to be arranged to
+  compensate for the overlapped area.
+
+  When using a \l{WebView} this property will be set automatically when the
+  keyboard is opened or closed.
+*/
+
+/*!
+  \qmlsignal WebView::contentOrientationChanged(Qt::ScreenOrientation orientation)
+  \brief This signal is emitted when the screen orientation of the rendered
+         content changes.
+*/
+
+/*!
+  \internal
+
+  \qmlsignal WebView::openUrlInNewWindow()
+  \brief This signal is emitted to indicate a request to open a new window.
+*/
+
+// ------------------------------------------------------ from QuickMozView.h:
+
+/*!
+  \qmlproperty bool WebView::privateMode
+  \brief Whether the webview is in private browsing mode
+
+  If private mode is enabled, history will not be stored in the device.
+
+  Note: the value for this property must be set when the webview is
+  initialized.  Any attempt to set the value of this property after the
+  webview has been instantiated will have no effect.
+*/
+
+/*!
+  \qmlproperty bool WebView::active
+  \brief Whether the webview is actively evaluating or rendering content
+
+  If a webview is not set as active, it will not display any content.
+*/
+
+/*!
+  \qmlproperty bool WebView::loaded
+  \brief Whether the webview content has finished loading successfully
+*/
+
+/*!
+  \qmlproperty Qt::ScreenOrientation WebView::orientation
+  \brief The orientation of the webview
+
+  Ordinarily, the value of this property will follow the orientation of the
+  (Silica) Page in which the webview is shown.
+*/
+
+/*!
+  \qmlproperty qreal WebView::viewportWidth
+  \brief The width of the webview viewport
+
+  Note that this will usually be the same as the width of the webview, but
+  in some cases clients may wish to set the viewport width explicitly
+  (e.g. to show some chrome at one edge or the other, without obscuring some
+  content underneath, while keeping the geometry of the webview item itself
+  unchanged).
+
+  \sa contentRect
+*/
+
+/*!
+  \qmlproperty qreal WebView::viewportHeight
+  \brief The height of the webview viewport
+
+  Note that this will usually be the same as the height of the webview, but
+  in some cases clients may wish to set the viewport height explicitly
+  (e.g. to show some chrome at one edge or the other, without obscuring some
+  content underneath, while keeping the geometry of the webview item itself
+  unchanged).
+
```

Comment:
Worth mentioning maybe that device orientation should be considered when adding a binding to this property.

By default viewportHeight follows screen height:
https://github.com/sailfishos/sailfish-components-webview/blob/master/import/webview/WebView.qml#L64

Reply:
done